x/mint/keeper/keeper.go

```diff
@@ -315,3 +344,96 @@ func (k Keeper) createDeveloperVestingModuleAccount(ctx sdk.Context, amount sdk.
 	}
 	return nil
 }
+
+// handleTruncationDelta estimates and distributes truncation delta from either mint module account or
+// developer vesting module account. Returns the total amount distributed from truncations.
+// If truncations are estimated to be less than one, persists them in store until the next epoch without
+// any distributions during the current epoch.
+// More on why this truncations handling is necessary: due to limitations of some SDK interfaces that operate on integers,
+// there are known truncation differences from the expected total epoch mint provisions.
+// To use these interfaces, we always round down to the nearest integer by truncating decimals.
+// As a result, it is possible to undermint. To mitigate that, we distribute any delta to the community pool.
+// The delta is calculated by subtracting the actual distributions from the given expected total distributions
+// and adding it to any left overs from the previous epoch. The left overs might be stemming from the inability
+// to distribute decimal truncations less than 1. As a result, we store them in the store until the next epoch.
+// These truncation distributions have eventual guarantees. That is, they are guaranteed to be distributed
+// eventually but not necessarily during the same epoch.
```

Comment:
nit: This technically isn't a guarantee right? Eventually emissions will become so low that the accumulator will never get past 1 right?

Reply:
Yeah that is fine, it was more me just thinking out loud haha, I'm cool with this guarantee